src/promptflow/promptflow/_constants.py

```diff
@@ -38,3 +38,12 @@ class AvailableIDE:
 
 USER_AGENT = "USER_AGENT"
 PF_USER_AGENT = "PF_USER_AGENT"
+
+CLI_PACKAGE_NAME = 'promptflow'
+CURRENT_VERSION = 'current_version'
+LATEST_VERSION = 'latest_version'
+LAST_HINT_TIME = 'last_hint_time'
+LAST_CHECK_TIME = 'last_check_time'
+PF_VERSION_CHECK = "pf_version_check.json"
+HINT_FREQUENCY_DAY = 7
```

Comment:
nit. frequency is not a good variable name **for me**: when I see frequency, I think there will be a cron job; but just go through the code, it works more like an interval for check; so my suggestion here is `HINT_INTERVAL` and `GET_PYPI_INTERVAL`

Reply:
it's my preference, feel free to adopt or keep it current.